hokusai/production.yml

```diff
@@ -298,9 +396,16 @@ spec:
             - partners:daily_digest
```

Comment:
question (non-blocking): Don't we need to replace `command` with `args` for the rake task?

Reply:
good catch. updated.